soc/nordic/nrf54h/gpd/Kconfig

```diff
@@ -6,6 +6,7 @@ config SOC_NRF54H20_GPD
 	imply NRFS
 	imply NRFS_GDPWR_SERVICE_ENABLED
 	select ONOFF
+	imply PINCTRL
```

Comment:
Why not `select` as in drivers (48dc82a81d3ceab63d47ca85cc90b1b88800ad4a)? GPD won't build without pinctrl, right?

Reply:
theoretically it builds without, its only if the application actually calls `nrf_gpd_retain_pins_set()` the user gets a linking error if not selected. Will change it to select though, pinctrl is probably always used anyway :)